db/schema.rb

```diff
@@ -525,6 +525,12 @@
     t.datetime "updated_at",                null: false
   end
 
+  create_table "featured_policies", force: :cascade do |t|
+    t.string  "policy_content_id", limit: 255
+    t.integer "ordering",          limit: 4
```

Comment:
It shouldn't ever be able to get in a null state due to the [before save](https://github.com/alphagov/whitehall/commit/06978b429f22c8ff78db3021047e69c22d9c4cc3#diff-1f693d12e26b3550ceb2198d1eaf824bR6) on the object adding a value, but have added the constraint.